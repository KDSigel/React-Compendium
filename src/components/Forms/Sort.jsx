import React from 'react'

export default function Sort({sortPoke, setSortPoke}) {
    return (
        <div>
            <select aria-label='sort' value={sortPoke}
            onChange={(e) => setSortPoke(e.target.value)}>
                <option value='asc'>ascending</option>
                <option value='desc'>descending</option>
            </select>
        </div>
    )
}
