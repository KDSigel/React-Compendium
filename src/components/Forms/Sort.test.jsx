import { screen, render } from "@testing-library/react";
import Sort from "./Sort";

it("testing my sort", () => {
  render(<Sort />);
  const sorted = screen.getAllByLabelText('sort');
  expect(sorted).toMatchSnapshot();
});
