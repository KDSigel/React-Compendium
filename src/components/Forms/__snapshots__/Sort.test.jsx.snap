// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing my sort 1`] = `
Array [
  <select
    aria-label="sort"
  >
    <option
      value="asc"
    >
      ascending
    </option>
    <option
      value="desc"
    >
      descending
    </option>
  </select>,
]
`;
